CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Starlette
+
+The Starlette team happily welcomes contributions. This document will help you get ready to contribute to Starlette!
+
+> **NOTE**: before writing any code, please first [open an issue] and discuss your ideas with the maintainers.
+
+## Setting up the repository
+
+1. Fork the repo.
+2. Clone your fork on your local computer: `git clone https://github.com/<username>/starlette.git`.
+3. Install Starlette locally and run the tests (see next sections).
+4. Create a branch for your work, e.g. `fix/some-bug`.
```

Comment:
I guess we should use the full `git checkout -b fix/some-bug` command here.  I don't particularly care about the two part 'fix/<name>' style, so perhaps just use `fix-some-bug` as the example branchname instead?

Reply:
Sure! I added a fix for this.